File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/trig/inverse/AtanZ.java
package net.fwitz.math.main.binary.complex.functions.trig.inverse;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class AtanZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("atan(z)")
                .computeFn(Complex::atan)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/trig/inverse/AtanZ.kt
package net.fwitz.math.main.binary.complex.functions.trig.inverse

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object AtanZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("atan(z)")
        .computeFn(Complex::atan)
        .render()
}